app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -576,6 +576,13 @@ fun EditActivityScreen(
                     ),
                 onClick = {
                   listActivityViewModel.deleteActivityById(activity?.uid ?: "")
+                  imageViewModel.removeAllActivityImages(
+                      activity?.uid ?: "",
+                      { Log.d("EditActivityScreen", "Images removed") },
+                      { error ->
+                        Log.e("EditActivityScreen", "Failed to remove images: ${error.message}")
+                      })
+
```

Comment:
That is good, the deletion function is called at the moment where you delete an activity